src/WebJobs.Script/Config/NameResolver.cs

```diff
@@ -2,17 +2,22 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System;
-using System.Configuration;
 using System.Globalization;
 
 namespace Microsoft.Azure.WebJobs.Script.Config
 {
-    internal class NameResolver : INameResolver
+    internal class NameResolver : DefaultNameResolver
```

Comment:
I can address that by simply reordering the checks in my NameResolver. See updated code.